src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -966,6 +972,45 @@ export default Vue.extend({
       videojs.registerComponent('fullWindowButton', fullWindowButton)
     },
 
+    createTheatreModeButton: function() {
+      if (!this.$parent.theatrePossible) {
+        return
+      }
+
+      const theatreModeActive = this.defaultTheatreMode ? ' vjs-icon-theatre-active' : ''
```

Comment:
Since the setting means "theatre mode active", should we name it `defaultTheatreModeActive`?
```suggestion
      const theatreModeActive = this.defaultTheatreModeActive ? ' vjs-icon-theatre-active' : ''
```

Reply:
It's the name of the variable as it's stored already. Not necessary IMO.

Reply:
OK I agreed